enhancements/split_helm_ansible_into_their_repos.md

```diff
@@ -0,0 +1,283 @@
+- I am an Ansible/Helm operator developer, I want to be able to use the command `make run`, so that I can run the project locally 
+- I am a maintainer/contributor, I want to be able to use the Operator type binary to run the CLI commands for the plugin, so that I can work with the project as standalone and I can use it to implement e2e test.
+
+#### Changelog for Operator Types
+
+- I am a maintainer/contributor, I want to use a Makefile target to generate the changelog and Migration Guide, so that I would spend low effort on the release process
+
+#### Tests for Operator Types
+
+- I am a maintainer/contributor, I want to use check the Operator Types covered with unit tests, so that I can ensure the code used to build based operator images 
+- I am a maintainer/contributor, I want to use check the Operator Types covered with e2e tests, so that I can ensure the projects that can be build with via any tool that will call its plugin
+
+### Risks and Mitigations
+
+Currently, when we start to support in SDK a new plugin version from upstream we can face a breaking change which will affects the specific SDK features such as for Scorecard and OLM. The same scenario might to be faced when we bump Ansible/Helm versions on SDK. 
+
+### Implementation Details/Notes/Constraints 
+
+**Details:**
+
+- The Operator Types (Ansible/Helm) binaries should be released from their repos and no longer via SDK
+- The operator based image implementation used on the Operator-Types (Ansible/Helm) should be public 
+- Ansible/Helm repositories should contain the same changelog feature that is provided in SDK.
+
+**Notes:**
+
+As a longer-term option solution, after the second phase of plugins are implemented, and its design be consolidated, we might be able to provide from upstream a base config plugin which could scaffold the `config/` directory as a common Makefile for any Operator Type be able to re-use it and perform its customizations as scaffold its specific languages type instead of Go.  SDK also could provide as lib its plugins for customizations to be done on top of it.
+
+This solution could be helpful to provide alternatives to developers quickly built their Operator Types such as to support [Argo](https://github.com/operator-framework/operator-sdk/issues/2497) and to allow the common needs to be re-used and be easily support new Operator Types such as [Hybrid Operators](https://github.com/operator-framework/operator-sdk/issues/670) or with other languages see [Meta: Operator SDK's in other languages #2745](https://github.com/operator-framework/operator-sdk/issues/2745), so that we could easily join these new Operator Types as incubator projects and define as pre-requirement following the same standards and workflows to work with the SDK features. However, it is definitely out of the scope of this EP. 
+
```

Comment:
Hi @estroz, 

I tried to clarify my ideas here. Besides these needs are not a Go of this EP I think it is important we have them in mind since they are ultimate goals for the projects. Otherwise, we might take decisions that will be harder to reach them in the future.